config/initializers/i18n.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+I18n.available_locales = [:fr]
+I18n.default_locale = :fr
+
+Rails.application.configure do
+  config.time_zone = "Europe/Paris"
+  config.i18n.raise_on_missing_translations = Rails.env.local?
+end
```

Comment:
Est-ce qu'on peut rester dans `config/application.rb` comme [le suggère la doc Rails](https://guides.rubyonrails.org/i18n.html#configure-the-i18n-module) ? Les setters sont renvoyés à `I18n` si jamais : https://github.com/rails/rails/blob/main/activesupport/lib/active_support/i18n_railtie.rb#L58.

Et pour ce qui est des défauts, je préfère qu'on ait `available_locales = %i[fr en]` avec un défaut sur `fr` que juste `fr` ? En se disant que pour les gems qui fournissent des traductions (genre Devise) ça permet d'avoir au moins ça ?